Makefile-package.mk

```diff
@@ -0,0 +1,51 @@
+PACKAGE_TYPES     ?= deb rpm
+PROJECT_NAME       = nri-$(INTEGRATION)
+BINS_DIR           = $(TARGET_DIR)/bin/linux_amd64
+SOURCE_DIR         = $(TARGET_DIR)/source
+PACKAGES_DIR       = $(TARGET_DIR)/packages
+VERSION           ?= 0.0.0
+RELEASE           ?= dev
+LICENSE            = "https://newrelic.com/terms (also see LICENSE.txt installed with this package)"
+VENDOR             = "New Relic, Inc."
+PACKAGER           = "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+PACKAGE_URL        = "https://www.newrelic.com/infrastructure"
+SUMMARY            = "New Relic Infrastructure $(INTEGRATION) Integrations"
```

Comment:
Nitpick (also for other repos): put "integrations" in singular.

Also for the `DESCRIPTION` variable, put in singular and mention NGINX.